monGARS/core/style_finetuning.py

```
@@ -0,0 +1,606 @@
+    PreTrainedTokenizerBase,
+    Trainer,
+    TrainingArguments,
+    default_data_collator,
+)
+
+try:
+    from peft import LoraConfig, PeftModel, get_peft_model
+except ImportError as exc:  # pragma: no cover - dependency missing at import time
+    raise RuntimeError(
+        "peft is required for style fine-tuning. Install the 'peft' package."
+    ) from exc
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)